_includes/decommissioning_checklist.md

```diff
@@ -0,0 +1,17 @@
+* [ ] Export any production data and put it into Google Drive, for [records retention](https://handbook.18f.gov/records-management/) purposes.
+* [ ] Ask in [#cg-support](https://gsa-tts.slack.com/messages/cg-support) for the cloud.gov organization to be deleted.
```

Comment:
Prefer cloud-gov-support@gsa.gov for org deletion requests (goes into ticketing system; less likely to lose track of the task).